src/Applications/LBAF.py

```diff
@@ -89,8 +89,8 @@ def __init__(self):
         self.time_sampler_parameters = []
 
         # Object communication graph time sampler type and parameters
-        self.weight_sampler_type = None
```

Comment:
Please change that as well in the `conf.yml` file so the description is accurate as well

Reply:
Sorry I see you changed it, I pulled the code and I don't have it locally :/

Reply:
Do you have it now?